packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1316,11 +1353,13 @@ mixin WidgetInspectorService {
           return false;
         }
         selection.currentElement = object;
+        developer.inspect(selection.currentElement);
```

Comment:
What does this inspect call do?

Reply:
calling `dart:developer` inspect on an object dispatches a message to all VM Service clients to inspect the object. For example, update the inspector to highlight the object or show a view of the object in a debugger or logging view.
It is a bug that this call to inspect existed for the renderObject case but not for this case.
This line is unrelated to the rest of the CL and I can move it to a separate CL.

Reply:
OK, great.  That clarifies things.  Thanks!